### PR TITLE
Hitesh's GCI-2017

### PR DESCRIPTION
**Pull request by Hr220903**
Hi, my name is Hitesh and this is my blog
https://hiteshsgci2017experience.wordpress.com/
https://fossasia.org/
image1

Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a Preview link and screenshot showning after and before the changes.
- [x] Images are `240 x 240` [w x h].
- [x] Included a description of change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`

### CODE REVIEW

**Review by isuruAb — COMMENTED** [review]
you didn't do any changes please check again

**Comment by supunTE** [comment]
Please refer README.md

**Review by nikit19 — CHANGES_REQUESTED** [review]
Please read the readme.md to see how to add your blog.

**Comment by nikit19** [comment]
Please resolve conflicts and make sure that the dimensions of the image is 240 x 240 pixels

**Review by nikit19 — APPROVED** [review]

**Comment by abishekvashok** [comment]
Please fix the conflicts

File: _data/blogs.yml [hunk 1]
```diff
@@ -198,9 +198,12 @@
   img: saarthakchaturvediinterview.jpg
   author_name: Saarthak Chaturvedi
 
+- url: https://hiteshsgci2017experience.wordpress.com/
+  title: Hitesh's GCI-2017 experience
+  img: image1.jpeg
```
Comment (anshumanv) [inline _data/blogs.yml]: Please rename image to exactly the file name of the image that you uploaded.

**Review by anshumanv — CHANGES_REQUESTED** [review]
Check comments.

**Review by isuruAb — COMMENTED** [review]
please resolve conflicts

**Review by harshit98 — APPROVED** [review]

**Pull request merged**